### ISSUE TITLE
Missing Alt Text

### ISSUE THREAD

**Issue by mgifford** [usds/ditap-curriculum-update#634]
You can find some missing alt text here:

https://github.com/search?q=repo%3Ausds%2Fditap-curriculum-update+%22%21%5B+%5D%22&type=code

https://github.com/search?q=repo%3Ausds%2Fditap-curriculum-update+%22%21%5B%5D%22&type=code

I ran across it here https://github.com/usds/ditap-curriculum-update/blob/main/3_Curriculum/3B_DITAP-Core-Curriculum/Module-2/Sprint-4/Conducting%20Effective%20Market%20Research.md but that wasn't the only space
